lib/matplotlib/tests/test_legend.py

```diff
@@ -1450,3 +1451,53 @@ def test_boxplot_labels():
     handles, labels = ax.get_legend_handles_labels()
     assert len(handles) == 0
     assert len(labels) == 0
+
+
+def test_boxplot_legend_labels():
+    # Test that legend entries are generated from boxplots. `label` is the new
+    # parameter for setting legend labels.
+    np.random.seed(19680801)
+    data = np.random.lognormal(size=(68, 4), mean=1.5, sigma=0.2)
+    labels = list('ABCD')
```

Comment:
```suggestion
```
Only used in `boxplot_stats()` but it's an optional parameter there and can be left out.